addons/payment_adyen/static/src/js/payment_form.js

```diff
@@ -56,7 +56,7 @@ odoo.define('payment_adyen.payment_form', require => {
         _dropinOnError: function (error) {
             this._displayError(
                 _t("Incorrect Payment Details"),
-                _t("Please verify your payment details.")
+                error.message.data.message
```

Comment:
hm... I thought we agreed on not showing the technical errors sent by providers to the customers, only log or send in chatter when necessary?
cc: @dalaOdoo

Reply:
I was not aware of that, if @dalaOdoo will confirm this then indeed I will go back to previous version.

Reply:
are we able to know what is the error ?
The customer should know the payment failed but the error message should contain information to understand how to correct. Here he will just check for card number, ... and nothing will change he will stay blocked and frustrated